packages/manager/modules/bm-server-components/src/ipmi/service.js

```diff
@@ -120,4 +97,76 @@ export default class BmServerComponentsIpmiService {
       })
       .then(({ data }) => data);
   }
+
+  createAndAssignNewCart() {
+    return this.$http
+      .post('/order/cart', {
+        ovhSubsidiary: this.user.ovhSubsidiary,
+      })
+      .then(({ data }) =>
+        this.$http.post(`/order/cart/${data.cartId}/assign`).then(() => ({
+          cartId: data.cartId,
+        })),
+      );
+  }
+
+  addKvmOptionToCart(cartId) {
+    return this.$http
+      .post(`/order/cart/${cartId}/eco`, {
+        duration: 'P1M',
+        planCode: KVM_PLAN_CODE,
+        pricingMode: 'default',
+        quantity: 1,
```

Comment:
**suggestion:** `duration`, `pricingMode` and `planCode` can be the arguments to the function. this way it can be reused.